src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/EditorTests.java

```diff
@@ -83,13 +82,16 @@ public void MercuryCuratedEditorTest_001_addAndSaveItemToFeaturedContent() {
     );
   }
 
-  @Test(groups = "MercuryCuratedEditorTest_002", enabled = false)
+  @Test(groups = "MercuryCuratedEditorTest_002")
   @Execute(onWikia = MercuryWikis.MERCURY_EMPTY_CC_EDITOR, asUser = User.STAFF)
   public void MercuryCuratedEditorTest_002_addAndSaveSection() {
     CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
     CuratedContentPageObject curatedContentPageObject = new CuratedContentPageObject(driver);
     EditorHomePageObject home = new EditorHomePageObject(driver);
 
+    curatedMainPagePageObject
+        .openArticleOnWikiByNameWithCbAndNoAds(wikiURL, MercurySubpages.ECC_MAIN_PAGE);
+
     Boolean result = !curatedMainPagePageObject.isCuratedContentVisible();
     PageObjectLogging.log(
         "Curated Content",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Curated Content" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)